Source/Orts.Simulation/Simulation/RollingStocks/MSTSLocomotive.cs

```diff
@@ -1653,6 +1653,7 @@ public override void InitializeMoving()
                 TrainControlSystem.InitializeMoving();
                 TrainBrakeController.InitializeMoving();
                 BrakeSystem.LocoInitializeMoving();
+                EngineBrakeController?.SetPercent(0);
```

Comment:
The MSTSBrakeController class has an InitializeMoving function with the following code:
`        public override void InitializeMoving()
        {
            NotchController.SetValue(0);
            NotchController.CurrentNotch = 0;
        }`
Shouldn't it be enough to call that function?

Reply:
```suggestion
                EngineBrakeController?.InitializeMoving();
```